zerver/webhooks/front/doc.md

```diff
@@ -2,36 +2,26 @@ Front lets you manage all of your communication channels in one place,
 and helps your team collaborate around every message. Follow these steps
 to recieve Front notifications without leaving Zulip!
 
-{!create-stream.md!}
+1. {!create-stream.md!}
 
-{!create-bot-construct-url.md!}
+2. {!create-bot-construct-url.md!}
 
-Then, log into your Front account, and do the following:
+3. Log into your Front account, and open the **Settings**.
 
-1. Go to the **Settings** of your organization.
-   ![](/static/images/integrations/front/001.png)
+4. On the **Integrations** tab, find and enable the **Webhooks**
+integration.
 
-2. Enable webhook notifications.
-   ![](/static/images/integrations/front/002.png)
-   ![](/static/images/integrations/front/003.png)
+5. Then, open the **Rules** tab, and add a new rule: pick the types of
+events for which you want to receive notifications, and set the URL you
+construsted for the Front bot as a target webhook.
```

Comment:
So this one should be:
```
1. Select the events you'd like to be notified about. Set the URL to the URL constructed above.
```

Reply:
Front allows to trigger many types of events apart from webhook notifications, so I decided to go with `Set the URL of the target webhook` to add a little clarity.